cni/pkg/ambient/informers.go

```diff
@@ -133,18 +133,25 @@ func (s *Server) Reconcile(input any) error {
 	event := input.(controllers.Event)
```

Comment:
We'd like to exclude the health check from redirection? It may be complex to filter out and current logic should be the same with ambient iptables mode and sidecar mode?

Reply:
This is the same with current ambient iptalbes mode and previous sidecar mode.  Traffic will be captured by sidecar(ztunnel)  and passthrough accordingly.

Reply:
We should filter all hostNetwork -> pod traffic which includes heathl check. This is the logic used in ambient iptables mode.

Reply:
Can you point us the current filter logic in iptalbes?

Reply:
`// cni/pkg/ambient/net.go
		newIptableRule(
			constants.TableMangle,
			constants.ChainZTunnelOutput,
			"--source", HostIP,
			"-j", "MARK",
			"--set-mark", constants.ConnSkipMask,
		),`
This rule makes HostNetwork-> pods traffic be skipped...
To simplify the logic in ebpf mode, we may skip the redirection to ztunnel by matching HostIP...
However, we should find a better way to retrieve the 'HostIP' used as the source IP for healthy check or other admin operation..

Reply:
We have exclude hostNetwork -> pod as well in ebpf